static_report/src/components/shared/Tables/TableList/SRTableListItem/SRTableListAssertionsSummary.tsx

```diff
@@ -0,0 +1,20 @@
+import { ColumnAssertionLabelWidget } from '../../../Widgets/ColumnAssertionLabelWidget';
+import { getAssertions } from '../../../../../utils/assertion';
+import { type AssertionTest } from '../../../../../sdlc/single-report-schema';
+
+export function SRTableListAssertionsSummary({
```

Comment:
Is there a comparison version of this component? (couldn't find it xD)

If it's a standalone, I'd suggest removing the `SR` or `CR`, where possible.

Also, If it's general enough, I think this belongs in `components/shared/Assertions/..`

Reply:
Good catch! I didn't notice, this component is redundant and can be replaced by another one.